tools/build.sh

```diff
@@ -113,9 +128,13 @@ LIST_CSS_TOPLEVEL="../../../node_modules/@nimiq/style/nimiq-style.min.css ../../
 # (since all urls are relative to request directories, we simply use the create request directory as the base)
 for url in $LIST_JS_COMMON; do
     cat src/request/create/$url >> dist/request/$JS_COMMON_BUNDLE
+    if [ "$KEYGUARD_ALLOWED_ORIGIN" != "" ]; then
+        replace_config_variable "CONFIG_ALLOWED_ORIGIN" "KEYGUARD_ALLOWED_ORIGIN" dist/request/$JS_COMMON_BUNDLE
+    fi
 done
 for url in $LIST_JS_TOPLEVEL; do
     cat src/request/create/$url >> dist/request/$JS_TOPLEVEL_BUNDLE
+    replace_config_variable "CONFIG_ALLOWED_ORIGIN" "KEYGUARD_ALLOWED_ORIGIN" dist/request/$JS_COMMON_BUNDLE
```

Comment:
In the replace above for the common bundle, you have a check if KEYGUARD_ALLOWED_ORIGIN is set and not empty. Why not here?

Reply:
Thanks, that other check became redundant as I introduced it generically in replace_config_variable().